ApplicationCode/ProjectDataModel/RimEclipseView.cpp

```diff
@@ -176,6 +178,10 @@ RimEclipseView::RimEclipseView()
     m_annotationCollection = new RimAnnotationInViewCollection;
     m_annotationCollection.uiCapability()->setUiHidden( true );
 
+    CAF_PDM_InitFieldNoDefault( &m_streamlineCollection, "StreamlineCollection", "Streamlines", "", "", "" );
+    m_streamlineCollection = new RimStreamlineInViewCollection();
```

Comment:
You can now use std::make_unique<RimStreamLineInViewCollection>() instead, since there is an assignment operator for the field that accepts a unique_ptr